src/main/scala/com/codacy/analysis/cli/command/CLIApp.scala

```diff
@@ -114,8 +114,13 @@ final case class Analyse(
   upload: Int @@ Counter = Tag.of(0),
   @ExtraName("p") @ValueDescription("The number of tools to run in parallel")
   parallel: Option[Int] = Option.empty,
+  @ValueDescription("The maximum number of issues allowed to pass the analysis")
+  maxAllowedIssues: Int = 0,
```

Comment:
It makes sense for me that this value is optional, instead of failing by default on any issue found.

Reply:
This is the same default as the cloud quality settings